src/app/core/entity-details/form/form-config.ts

```diff
@@ -0,0 +1,16 @@
+import { ColumnConfig } from "../../common-components/entity-subrecord/entity-subrecord/entity-subrecord-config";
+
+/**
+ * The (possibly abbreviated) configuration for a "FormComponent", as it is stored in the config file.
+ */
+export interface FormConfig {
+  fieldGroups: FieldGroup[];
+}
```

Comment:
What is this interface there for?

Reply:
This is slightly trivial at the moment but defines the format allowed in config doc (similar to `EntityDetailsConfig` and others). I think this is useful, even though we don't actively validate config formats usually.
Should we make the code comment clearer, somehow?

Reply:
ok, removed as discussed